core/src/main/codegen/templates/Parser.jj

```diff
@@ -1335,7 +1335,7 @@ SqlSelect SqlSelect() :
 }
 {
     <SELECT> { s = span(); }
-    [ <HINT_BEG> AddHint(hints) ( <COMMA> AddHint(hints) )* <COMMENT_END> ]
+    [ <HINT_BEG> (AddHint(hints))? ( <COMMA> AddHint(hints) )* <COMMENT_END> ]
```

Comment:
I think hint start with COMMA is illegal syntax,  but your change will allow it to pass during parsing.
For example `select /*+ , hint1 */`

Reply:
> I think hint start with COMMA is illegal syntax, but your change will allow it to pass during parsing. For example `select /*+ , hint1 */`

Hi @macroguo-ghy ,thanks for your review,I will take a look for checking.